src/widget/woverview.cpp

```diff
@@ -202,6 +203,10 @@ void WOverview::setup(const QDomNode& node, const SkinContext& context) {
 void WOverview::onConnectedControlChanged(double dParameter, double dValue) {
     // this is connected via skin to "playposition"
     Q_UNUSED(dValue);
+    if (m_bLeftClickDragging) {
```

Comment:
This breaks the time ruler while dragging. The time between the mouse position and the play position does not get updated as the play position moves.